src/protocols/ethereum/components/TransferReview.vue

```diff
@@ -102,11 +102,10 @@ export default defineComponent({
       loading.value = true;
       let actionResult;
       const lastActiveEthAccount = getLastActiveProtocolAccount(PROTOCOLS.ethereum);
-
       try {
         if (!isSelectedAssetEthCoin.value) {
           actionResult = await ethAdapter.transferToken?.(
-            Number(amount),
```

Comment:
This could force the amount to be rounded which means that it could happen that user was trying to send `1.5 LINK` when he only had `1.4999999999999 LINK`